### PR TITLE
Improvement: Content/Post timeline html optimization.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Removed classes listed here - https://docs.google.com/document/d/1ZNsQoG4eEtac9_bxWwwt6XBkOSdfarQ3D9vaulpcss0/edit

### Screenshots

- https://share.getcloudapp.com/04u5nwN2

### Types of changes

- Improvement

### How has this been tested?

- Case 1 - Backward compatibility.
- Case 2 - By checking all Settings.
- Case 3 - Special test case - Check all Orientation, Stack-On, Alignment options with left/right orientation.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]
@AbhijitNage123 Please check in editor - https://share.bsf.io/o0u7Aw8Z

Tried Regenerate Assets too.

**Comment by AbhijitNage123** [comment]
> @AbhijitNage123 Please check in editor - https://share.bsf.io/o0u7Aw8Z
> 
> Tried Regenerate Assets too.

@RajkiranBagal could you please delete custom-style-blocks.css and check again. I'm not able to replicate the same at my end.

**Review by RajkiranBagal — APPROVED** [review]

**Comment by akshaylakh** [comment]
@AbhijitNage123 - 
Post Timeline > Console error -  https://share.bsf.io/Z4ur0Prw 
Post Timeline > Spacing > Content Spacing not working
Post/Content Timeline > Arrow disappeeared for center orientation - https://share.bsf.io/5zu9z7jR
Content timeline - left/ right orientation > layout break -  https://share.bsf.io/qGuReyYQ

**Comment by akshaylakh** [comment]
@AbhijitNage123 -  Post Timeline > Console error - https://share.bsf.io/Z4ur0Prw
Others are working fine

**Comment by supriyajjadhav** [comment]
On mobile view date element also align on center Please update media query - https://share.getcloudapp.com/Qwu2ggyp

**Comment by AbhijitNage123** [comment]
> On mobile view date element also align on center Please update media query - https://share.getcloudapp.com/Qwu2ggyp

This issue is fixed. Working fine on Mobile. See here - https://share.getcloudapp.com/7KuADGDj

**Comment by akshaylakh** [comment]
Post timeline - Block error - [img](https://share.bsf.io/6quERPZq)
Content timeline breaking on front end -  [rec](https://share.bsf.io/GGuEXrlp)

**Comment by AbhijitNage123** [comment]
> Post timeline - Block error - [img](https://share.bsf.io/6quERPZq) Content timeline breaking on front end - [rec](https://share.bsf.io/GGuEXrlp)

Fixed

**Comment by akshaylakh** [comment]
working fine

File: src/blocks/timeline/content-timeline/edit.js [hunk 1]
```diff
@@ -78,84 +78,84 @@ const ContentTimelineComponent = ( props ) => {
...
+		 if (
+			 null ===
+			 select( 'core/block-editor' ).getBlocksByClientId(
+				 props.clientId
+				 )[ 0 ]
+				 ) {
+					 return;
+					}
+					select( 'core/block-editor' )
 			.getBlocksByClientId( props.clientId )[ 0 ]
```
Comment (sandeshjangam) [inline src/blocks/timeline/content-timeline/edit.js:93]: JS formatting?

Comment (AbhijitNage123) [reply src/blocks/timeline/content-timeline/edit.js]: Updated

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**